source/reference/mongosync/mongosync-behavior.txt

```diff
@@ -179,6 +179,17 @@ primary shard to each database by means of a round-robin.
    restart the migration from the start. For more information, see
    :ref:`c2c-sharded-limitations`. 
 
+Embedded Config Server Shards
+'''''''''''''''''''''''''''''
+
+Starting in 8.0, MongoDB introduces support for embedded config server shards.
+
+``mongosync`` supports sync from dedicated config server sharded clusters to 
+embedded config server sharded clusters and vice versa. Additionally, ``mongosync`` 
+supports sync from replica sets to embedded config sharded clusters.
+
+For more information, see :ref:`config-shard-concept`. 
```

Comment:
[nit] Given the context of users being on this page for `mongosync` behavior, it might be a bit misleading to say "for more information..." since users might expect to learn more about syncing from a config server cluster to an embedded config server. It might be better to either be specific with "To learn more about embedded config servers, see :ref:`config-shard-concept`" or hyperlink "embedded config server shards" on line 185 to the page.